modules/navier_stokes/test/tests/finite_volume/ins/solidification/tests

```diff
@@ -13,6 +13,7 @@
     input = 'pipe_solidification.i'
     exodiff = 'pipe_solidification_out.e'
     requirement = 'The system shall be able to solve solidification with advection in a pipe using the finite volume method.'
+    rel_err = 1.0E-2
```

Comment:
It's very difficult to match the friction factor in this specific test. The documentation says that the INSFVMushyPorousFrictionFunctorMaterial is compatible with [INSFVMomentumFriction](https://mooseframework.inl.gov/source/fvkernels/INSFVMomentumFriction.html) which multiplies the incoming Forchheimer coefficient by the velocity magnitude; it is incompatible with [PINSFVMomentumFriction](https://mooseframework.inl.gov/source/fvkernels/PINSFVMomentumFriction.html) which assumes the incoming Forchheimer coefficient already includes multiplication by the velocity magnitude. I can try to do something similar to what i did in the other test cases but it will get very ugly. Otherwise i can regold.

Reply:
there is a solidification example so I think we can do something ugly with a comment in the test

Reply:
fixed it